test-e2e/global-teardown.ts

```diff
@@ -0,0 +1,7 @@
+import { type Config } from '@playwright/test'
+import { $ } from 'execa'
+import { kuboRepoDir } from './fixtures/load-kubo-fixtures.js'
+
+export default async function globalTeardown (config: Config): Promise<void> {
+  await $`rm -rf ${kuboRepoDir}`
```

Comment:
I thought about pushing up the kubo repo just to simplify things, but that feels bad.